File: src/main/java/com/redit/db/Expense.java
package com.redit.db;

import com.j256.ormlite.field.DatabaseField;
import com.j256.ormlite.table.DatabaseTable;

/**
 * Created By : Lalit The Coder of house Umbarkar, First of his name, Khalasar of UAC, King in the West.
 * Created On : 21 Apr 2019
 * Organisation: CustomerXPs Software Private Ltd.
 */
@DatabaseTable(tableName = "expense")
public class Expense {

    @DatabaseField(generatedId = true)
    public int id;

    @DatabaseField(columnName = "name", canBeNull = false)
    public String title;

    @DatabaseField(columnName = "is_approved", canBeNull = false)
    public Boolean isApproved;

    @DatabaseField(foreign = true, columnName = "manager")
    public Employee manager;

}

File: ui/angular/app/models/expense.ts
/**
 * Created By : Lalit
 * Created On : 5/1/19
 * Organisation: CustomerXPs Software Private Ltd.
 */
import {EmployeeSummary} from "./user";

export class Expense {

    public id: number;
    public title: string;
    public isApproved: boolean;
    public amount: number;
    public manager: EmployeeSummary;

}

